playbooks/roles/mattermail/templates/mattermail.service.j2

```diff
@@ -11,6 +11,9 @@ Restart=always
 RestartSec=10
 User=mattermail
 Group=mattermail
+# Mattermail seems to be leaking memory, so we let systemd kill it if it uses too much memory.
+MemoryHigh=120M
+MemoryMax=150M
```

Comment:
Just to confirm, it'll get restarted automatically by systemd right, or will it just sit dead in case it got OOM killed?

Reply:
Sounds good.